apps/web/src/components/Hero/Hero.stories.tsx

```diff
@@ -38,25 +39,138 @@ export default {
         "dark iap": allModes["dark iap desktop"],
       },
     },
+    defaultPath: {
+      path: "/:index",
+      initialEntries: [`/one`],
+    },
   },
 } as Meta<typeof Hero>;
 
 const Template: StoryFn<typeof Hero> = (args) => <Hero {...args} />;
 
-export const Default = Template.bind({});
+export const ButtonsAndTabs = Template.bind({});
+ButtonsAndTabs.args = {
+  buttonLinks: [
+    { icon: UserPlusIcon, text: "Hello", url: "#one", color: "quinary" },
+    {
+      icon: AcademicCapIcon,
+      text: "Hello hello",
+      url: "#two",
+      color: "primary",
+    },
+    {
+      icon: ChatBubbleBottomCenterIcon,
+      text: "And goodbye",
+      url: "#two",
+      color: "tertiary",
+    },
+  ],
+  navTabs: [
+    { url: "/one", label: "Tab One" },
+    { url: "/two", label: "Tab Two" },
+    { url: "/three", label: "Tab Three" },
+    { url: "/four", label: "Tab Four" },
+    { url: "/five", label: "Tab Five" },
+  ],
+};
+
+export const Buttons = Template.bind({});
+Buttons.args = {
+  buttonLinks: [
+    {
+      icon: UserPlusIcon,
+      text: "Hello",
+      url: "#one",
+      color: "secondary",
+    },
+    {
+      icon: AcademicCapIcon,
+      text: "Hello hello",
+      url: "#two",
+      color: "warning",
+    },
+    {
+      icon: ChatBubbleBottomCenterIcon,
+      text: "And goodbye",
+      url: "#two",
+      color: "error",
+    },
+  ],
+};
+
+export const Tabs = Template.bind({});
+Tabs.args = {
+  navTabs: [
+    { url: "/one", label: "Tab One" },
+    { url: "/two", label: "Tab Two" },
+    { url: "/three", label: "Tab Three" },
+    { url: "/four", label: "Tab Four" },
+    { url: "/five", label: "Tab Five" },
+  ],
+};
+
+export const NeitherButtonsOrTabs = Template.bind({});
+NeitherButtonsOrTabs.args = {};
+
+export const ButtonsAndTabsWithCentering = Template.bind({});
+ButtonsAndTabsWithCentering.args = {
+  buttonLinks: [
+    { icon: UserPlusIcon, text: "Hello", url: "#one", color: "primary" },
+    {
+      icon: AcademicCapIcon,
+      text: "Hello hello",
+      url: "#two",
+      color: "secondary",
+    },
+    {
+      icon: ChatBubbleBottomCenterIcon,
+      text: "And goodbye",
+      url: "#two",
+      color: "tertiary",
+    },
+  ],
+  navTabs: [
+    { url: "/one", label: "Tab One" },
+    { url: "/two", label: "Tab Two" },
+    { url: "/three", label: "Tab Three" },
+    { url: "/four", label: "Tab Four" },
+    { url: "/five", label: "Tab Five" },
+  ],
+  centered: true,
+};
+
+export const ImageProvided = Template.bind({});
```

Comment:
I don't think the image is working.  The size looks really funny above mobile width.

Reply:
Oh I can fix this if you haven't already started on it @vd1992?

Reply:
If you got an idea please feel free to help get this done ASAP 😛

Reply:
Hopefully looks good in [99997c9](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11713/commits/99997c914a40172e1286a086a4443bb256e16441)

Reply:
Oh, it looks so good!  :heart_eyes:   I think the positioning is a little off from a width of 768px to 1080px, though.  The image just slides off the right side.  Smaller and greater than that range looks perfect, though.

https://github.com/user-attachments/assets/39ff4f63-1a7e-48a4-a7be-703029b06a42

Reply:
Yeah that could be improved, alright how about this https://github.com/GCTC-NTGC/gc-digital-talent/pull/11713/commits/e511003cfa6b25e8a24626b715af530293061f1c